frontends/web/src/components/aopp/vasp.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, JSX, RenderableProps } from 'preact';
+import * as styles from './vasp.css';
+import TwentyOneAnalyticsLogo from '../../assets/exchanges/logos/21analytics.png';
+import BitcoinSuisseLogo from '../../assets/exchanges/logos/bitcoin_suisse.png';
+import BittrLogo from '../../assets/exchanges/logos/bittr.png';
+import BityLogo from '../../assets/exchanges/logos/bity.png';
+import PocketBitcoinLogo from '../../assets/exchanges/logos/pocketbitcoin.svg';
+
+interface VASPProps {
+    fallback?: JSX.Element;
+    hostname: string;
+    prominent?: boolean;
+    withLogoText?: string;
+}
+
+const VASPLogoMap = {
+    'testing.21analytics.ch': TwentyOneAnalyticsLogo,
```

Comment:
i only see a login in this site - is this the right domain?

i know of:

https://testing.aopp.group/ and https://demo.aopp.group/

Reply:
Since today it's on and off a login, lets support testing.aopp.group and demo.aopp.group too, but should we usethe 21 logo or the AOPP logo?